cmd/txtar-addmod/addmod.go

```diff
@@ -133,12 +149,19 @@ func main1() int {
 		if !strings.Contains(arg, "@") {
 			title += "@" + vers
 		}
-		a.Comment = []byte(fmt.Sprintf("module %s\n\n", title))
+		dir = filepath.Clean(dir)
+		modDir := strings.Replace(path, "/", "_", -1)+"_"+vers
```

Comment:
I wonder whether we should extract this "logic"; it's repeated in `goproxytest`.

Reply:
Yeah, probably. Perhaps not right now - this is an existing issue and it's not really a pressing one (we can't change the convention anyway, as too many existing txtar files use it)